packages/react-log-viewer/src/LogViewer/LogViewer.tsx

```diff
@@ -28,6 +28,8 @@ interface LogViewerProps {
   toolbar?: React.ReactNode;
   /** Content displayed while the log viewer is loading */
   loadingContent?: React.ReactNode;
+  /** Flag indicating that log viewer is dark themed */
+  isDarkTheme?: boolean;
```

Comment:
I wonder if we should follow other components like pageSidebar and Nav 
` theme?: 'dark' | 'light';`